databricks-kube/src/traits/synced_api_resource.rs

```diff
@@ -417,6 +445,20 @@ pub trait SyncedAPIResource<TAPIType: 'static, TRestConfig: Sync + Send + Clone>
         )
     }
 
+    fn remote_drifted(
+        &self,
+        rhs: Box<TAPIType>,
+    ) -> bool 
+    where 
+        TAPIType: PartialEq,
+        TAPIType: From<Self>,
+        Self: Sized,
+        Self: Clone,
+    {
+        let self_as_api: TAPIType = self.clone().into();
+        self_as_api != *rhs
+    }
+
```

Comment:
Perhaps we don't even need this function if we're just doing a check on `if latest_remote != kube_as_api`